src/dispatch.iced

```diff
@@ -7,16 +7,16 @@ toBuffer = require('typedarray-to-buffer')
 
 iced = require('./iced').runtime
 
-COMPRESSION_TYPE_NONE = 0
-COMPRESSION_TYPE_GZIP = 1
+exports.COMPRESSION_TYPE_NONE = 0
```

Comment:
```suggestion
exports.COMPRESSION_TYPE_NONE = COMPRESSION_TYPE_NONE = 0
```

and then you don't need to have `exports` throughout this file

Reply:
I was looking for something like this, thanks!